lib/models/project.js

```diff
@@ -659,15 +659,20 @@ class Project {
     let ui = ensureUI(_ui);
 
     let directory = findupPath(pathName);
+    let pkg = fs.readJsonSync(path.join(directory, 'package.json'));
     logger.info('found package.json at %s', directory);
 
+    // allow `package.json` files to specify where the actual project lives
+    if (pkg && pkg.ember && typeof pkg.ember.projectRoot === 'string') {
```

Comment:
Should it be `pkg.ember` or `pkg.ember-addon` (where most of the other related config options are, like `configPath`)?

Reply:
@rwjblue yeah, this seems good; I'll update